ratelimit.c

```diff
@@ -115,6 +115,7 @@ int RaterLimit_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv,
       RedisModule_CloseKey(key);
       return RedisModule_ReplyWithError(ctx, "ERR invalid stored rater");
```

Comment:
@zozowell I just noticed there still might be an issue in case of error. We need to call the ` RedisModule_FreeString(ctx, tat_str);` function before the return as well.

Reply:
Oh yeah @edufelipe, sorry for missing this branch, I'm on it.